src/utils/host/translate/auto-translation.ts

```diff
@@ -1,13 +1,59 @@
 import type { Config } from '@/types/config/config'
 import { getFinalSourceCode } from '@/utils/config/languages'
 
+/**
```

Comment:
1. we don't like comment in code, because we believe good code explain themselves and self-documented
2. If you feel we have to add the comment somewhere, use English please